src/native/src/3rdparty/glad/include/KHR/khrplatform.h

```diff
@@ -26,7 +26,7 @@
 
```

Comment:
I probably didnt need to, but the main reason was I saw you had some path changes in its headers. And just rather than figure out where things should have gone, i just regenerated it instead. Did you have any specific changes ontop of it that you needed?